<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleSelection</class>
 <widget class="QWidget" name="RoleSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Role selection</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>581</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>bilibili</string>
    </attribute>
    <widget class="QPushButton" name="btn_22">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lab_22">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>25</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>22</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>碧蓝航线</string>
    </attribute>
    <widget class="QFrame" name="frame_Chongying">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>0</y>
       <width>580</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea_Chongying">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>20</y>
        <width>578</width>
        <height>280</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>576</width>
         <height>278</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>278</height>
        </size>
       </property>
       <widget class="QLabel" name="lab_biruishuiyi">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>54</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>比叡睡衣</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_biruishuiyi">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_niaohai">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>30</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>鸟海</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_niaohai">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_xili">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>70</y>
          <width>25</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>夕立</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_xili">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_nengdai">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_nengdai">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>70</y>
          <width>30</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>能代</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_Huangjia">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>0</y>
       <width>580</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea_huangjia">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>20</y>
        <width>578</width>
        <height>280</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>576</width>
         <height>278</height>
        </rect>
       </property>
       <widget class="QLabel" name="lab_kulasuo">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>50</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>库拉索</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_kulasuo">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_heitaiziSZ">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>70</y>
          <width>60</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>黑太子水着</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_heitaizi">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_heitaizi">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>70</y>
          <width>40</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>黑太子</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_heitaiziSZ">
        <property name="geometry">
         <rect>
          <x>165</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_TLXlifu">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>70</y>
          <width>60</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>天狼星礼服</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_TLXlifu">
        <property name="geometry">
         <rect>
          <x>245</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_ALBaiYing">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>302</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ALChongYing">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>302</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ALHuangJia">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>302</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ALTieXue">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>302</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ALUpPage">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>302</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ALNextPage">
     <property name="geometry">
      <rect>
       <x>355</x>
       <y>302</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_TieXue">
     <property name="geometry">
      <rect>
       <x>1740</x>
       <y>0</y>
       <width>580</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea_TieXue">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>20</y>
        <width>578</width>
        <height>280</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>576</width>
         <height>278</height>
        </rect>
       </property>
       <widget class="QPushButton" name="btn_dadi">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_dadi">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>30</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>大帝</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_dadi_h">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_dadi_h">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>35</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>大帝H</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_BaiYing">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>580</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea_BaiYing">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>20</y>
        <width>578</width>
        <height>280</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>576</width>
         <height>278</height>
        </rect>
       </property>
       <widget class="QPushButton" name="btn_huonvlulu">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>54</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>火奴鲁鲁</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>炼铜达人</string>
    </attribute>
    <widget class="QLabel" name="lab_liantong1">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>70</y>
       <width>25</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>小红</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_liantong1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>VOCALOID</string>
    </attribute>
    <widget class="QFrame" name="frame_LuoTianYi">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>580</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea_LuoTianYi">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>20</y>
        <width>578</width>
        <height>280</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>576</width>
         <height>278</height>
        </rect>
       </property>
       <widget class="QPushButton" name="btn_luotianyi01">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_luotian01">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>54</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>洛天依01</string>
        </property>
       </widget>
       <widget class="QLabel" name="lab_luotian02">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>54</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>洛天依02</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_luotianyi02">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_luotian03">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>70</y>
          <width>54</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>洛天依03</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_luotianyi03">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_luotian04">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>70</y>
          <width>54</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>洛天依04</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_luotianyi04">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lab_luotian05">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>70</y>
          <width>54</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>洛天依05</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_luotianyi05">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>0</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>VirtuaReal</string>
    </attribute>
    <widget class="QPushButton" name="btn_yousa">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lab_yousa">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>54</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>泠鸢yousa</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Hololive</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择角色：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Determine">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_RoleName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_RoleHeadImage">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
